lib/edamam_api_wrapper.rb

```diff
@@ -0,0 +1,53 @@
+require 'httparty'
+class EdamamApiWrapper
+  BASE_URL = "https://api.edamam.com/"
+  ID = ENV["APP_ID"]
+  KEY = ENV["APP_KEY"]
+
+  def self.list_recipes(ingredients)
+    url = BASE_URL + "search?" + "q=#{ingredients}" + "&app_id=#{ID}&app_key=#{KEY}" + "&to=80&excluded=BANANAS&excluded=BANANA&excluded=pl%C3%A1tano&excluded=onions&excluded=onion&excluded=mushrooms&excluded=avocado"
+    #add encode URL??
+    data = HTTParty.get(url)
+    recipe_list = []
+    if data["hits"].nil? || data["hits"] == []
+      return nil
+    else
+      data["hits"].each do |recipe_data|
+        recipe_list << create_recipe(recipe_data["recipe"])
+      end
+      return recipe_list
+    end
+
+  end
+
+  def self.find_recipe(id)
+    url = BASE_URL + "search?" + "r=http%3A%2F%2Fwww.edamam.com%2Fontologies%2Fedamam.owl%23recipe_#{id}" + "&app_id=#{ID}&app_key=#{KEY}"
```

Comment:
Minor style thing, you have one variable called `id` and another called `ID`, could be slightly confusing.